tests/Fixtures.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+#include "UsgsAstroFramePlugin.h"
+#include "UsgsAstroFrameSensorModel.h"
+
+#include <json.hpp>
+
+#include <sstream>
+#include <fstream>
+
+#include <gtest/gtest.h>
+
+using json = nlohmann::json;
+
+// Should this be positioned somehwere in the public API?
```

Comment:
The public API is the CSM API so we can't put utility functions like this out in this library. This will have to go into a utility library like the SensorUtils library.